code/tINIT/getAATripletReactions.m

```diff
@@ -0,0 +1,69 @@
+function rxns=getAATripletReactions(model, onlyRxnsWithoutGPRs)
+% getAATripletReactions
+%   Finds all reactions that work with amino acid triplets (or doublets). These are 
+%   not very useful in the model, and removing them simplifies the problem.
+%
+%   model               The model to be examined.
+%   onlyRxnsWithoutGPRs If true, the function only gets the rxns without GPRs
+%
+%   rxns                The resulting reactions
+%
+%   Usage: rxns=getAATripletReactions(model, onlyRxnsWithoutGPRs)
+%
+
+aminoAcidsCap = {'Alanine';'Arginine';'Asparagine';'Aspartate';'Cysteine';'Glutamine';'Glutamate';'Glycine';'Histidine';'Isoleucine';'Leucine';'Lysine';'Methionine';'Metheonine';'Phenylalanine';'Proline';'Serine';'Threonine';'Tryptophan';'Tyrosine';'Valine'};
```

Comment:
Not sure what `Metheonine` is, but it can probably be removed (should only be 20 AAs)

Reply:
So, the problem is that there are reactions where the amino acids are misspelled - that's why I have added those weird ones :)

Reply:
How about submitting another PR that fixes the misspelling, and merging that one first?

Reply:
Yeah sorry, I should have actually looked into it further before commenting.

Reply:
Is there a reason to not code in [all 22](https://en.wikipedia.org/wiki/Essential_amino_acid#Essentiality_in_humans)?

Reply:
So, the point of this function is to identify reactions that convert amino acid triplets back and forth, i.e. chains of three amino acids <=> three separate amino acids. It is a measure of optimization - these reactions are pretty pointless in most practical uses with tINIT. It is like 700 reactions or something like that. So I only added the amino acids that help identify those reactions.